File: ios/tests_class_sel_imp/web/Cattle.js
function Cattle()
{
	this.legsCount = null;
}

Cattle.prototype.saySomething = function saySomething()
{
	console.log("Hello, I am a cattle, I have " + this.legsCount + " legs.");
}

Cattle.prototype.setLegsCount = function setLegsCount(count)
{
	this.legsCount = count;
}

File: ios/06-NSObject/06-NSObject/Cattle.m
//
//  Cattle.m
//  tests_class_sel_imp
//
//  Created by 裴超 on 13-5-8.
//  Copyright (c) 2013年 裴超. All rights reserved.
//

#import "Cattle.h"

@implementation Cattle

-(void) saySomething
{
	NSLog(@"Hello, I am a cattle, I have %d legs.", legsCount);
}

-(void) setLegsCount:(int)count
{
	legsCount = count;
}

@end
